x-pack/plugins/maps/public/connected_components/timeslider/timeslider.tsx

```diff
@@ -182,6 +183,22 @@ class KeyedTimeslider extends Component<Props, State> {
             <EuiText size="s">{prettyPrintTimeslice(this.state.timeslice)}</EuiText>
           </div>
 
+          <EuiButtonIcon
+            onClick={() => {
+              this.props.updateGlobalTimeRange(this.state.timeslice);
+            }}
+            iconType="calendar"
+            display="fill"
```

Comment:
Can you remove the display property so the button looks like the below screen shot. That way, it will not stand out so much.

<img width="300" alt="Screen Shot 2022-01-12 at 9 10 02 AM" src="https://user-images.githubusercontent.com/373691/149155968-1ee52ab2-f07a-4ed6-adfa-05232cbe05f2.png">